src/molecules/modal/Modal.stories.js

```diff
@@ -0,0 +1,256 @@
+      }
+    },
+    data: () => ({ value: null }),
+    methods: {
+      showModal () {
+        this.$refs.modalDefault.show('default')
+      }
+    },
+    template: `
+      <div>
+        <a-button @click.native="showModal">
+          Modal button
+        </a-button>
+        <a-modal
+          name="default"
+          :heading="headingKnob"
+          ref="modalDefault"
+        >
+          <p style="margin-bottom: 8px;">{{ contentKnob }}</p>
+          <a-input-wrapper
+            input-id="field_id"
+            class="a-input-wrapper--inline"
+            label="Inline text"
+          >
+            <a-input
+              id="field_id"
+              type="text"
+              v-model="value"
+              placeholder="Test focus trap"
+            />
+          </a-input-wrapper>
+         </a-modal>
+      </div>
+    `
+  }))
+  .add('Without close icon', () => ({
+    components: {
+      AButton,
+      AModal,
+      AInput,
+      AInputWrapper
+    },
+    props: {
+      headingKnob: {
+        default: text('Modal heading', 'Modal heading')
+      },
+      contentKnob: {
+        default: text('Content', 'Lorem ipsum dolor sit amet, consectetur adipLorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.iscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.')
+      }
+    },
+    data: () => ({ value: null }),
+    methods: {
+      showModal () {
+        this.$refs.modalDefault.show('default')
+      }
+    },
+    template: `
+      <div>
+        <a-button @click.native="showModal">
+          Modal button
+        </a-button>
+        <a-modal
+          name="default"
+          :heading="headingKnob"
+          :closeButton="false"
+          ref="modalDefault"
+        >
+          <p style="margin-bottom: 8px;">{{ contentKnob }}</p>
+          <a-input-wrapper
+            input-id="field_id"
+            class="a-input-wrapper--inline"
+            label="Inline text"
+          >
+            <a-input
+              id="field_id"
+              type="text"
+              v-model="value"
+              placeholder="Test focus trap"
+            />
+          </a-input-wrapper>
+         </a-modal>
+      </div>
+    `
+  }))
+  .add('Without header', () => ({
+    components: {
+      AButton,
+      AModal,
+      AInput,
+      AInputWrapper
+    },
+    props: {
+      headingKnob: {
```

Comment:
This knob is not needed for this story 'without header'

Reply:
Damn, you're right - thanks!